This patch fixes a bug in symtabAPI over finding inlined subroutine
ranges that are a list of ranges rather than a single interval.
Apply the patch with:

cd dyninst-4fb1f52
patch -p1 <this-patch


diff --git a/symtabAPI/src/dwarfWalker.C b/symtabAPI/src/dwarfWalker.C
index d25af63..f794109 100644
--- a/symtabAPI/src/dwarfWalker.C
+++ b/symtabAPI/src/dwarfWalker.C
@@ -628,7 +628,7 @@ bool DwarfWalker::parseRangeTypes() {
       bool done = false;
       for (unsigned i = 0; i < ranges_length && !done; i++) {
          Dwarf_Ranges *cur = ranges + i;
-         Address cur_base = 0;
+         Address cur_base = modLow;
          switch (cur->dwr_type) {
             case DW_RANGES_ENTRY: {
                Address low = cur->dwr_addr1 + cur_base;
